hnn_core/viz.py

```diff
@@ -1062,3 +1062,57 @@ def _onclick(event):
 
     plt_show(show)
     return ax.get_figure()
+
+
+def _plot_cell(ax, cell_type=None, show=True):
+    """Plot the cell morphology of a specific cell type
+
+    parameters
+    ----------
+    cell_type : instance of net.cell_type[]
```

Comment:
This doesn't really specify the object being passed to the function since `net.cell_types` is a dictionary: 
https://github.com/jonescompneurolab/hnn-core/blob/9850f20a65c1e892a660f52b63aa861de3c2f429/hnn_core/network.py#L291

Is the goal to just pass the cell name, or the cell object to the function? If it's the latter, this would be an instance of the `Cell` class:
https://github.com/jonescompneurolab/hnn-core/blob/9850f20a65c1e892a660f52b63aa861de3c2f429/hnn_core/cell.py#L220